DuckDuckGo/Preferences/View/PreferencesVPNView.swift

```diff
@@ -55,9 +56,9 @@ extension Preferences {
                 }
                 .padding(.bottom, 12)
 
-                // SECTION: Excluded Sites
+                if model.exclusionsAvailable && !model.appExclusionsFeatureEnabled {
```

Comment:
If exclusions are available to the current platform, and app exclusions are disabled... show legacy UI.

Reply:
One comment about naming boolean properties - I'm a fan of prefixing them with `is` or `has` to make it clear that they are boolean. I see this is being followed elsewhere even in this file (`isValidDNSServers`, `isSheetPresented`), so just a note because it would be great to maintain consistency.

Reply:
Adjusted the naming and simplified the code a bit.